backend/users/tasks.py

```diff
@@ -0,0 +1,21 @@
+from celery import shared_task
+import requests
+from locations.models import LocationModel
+
+
+@shared_task
```

Comment:
Instead of shared task, A regular task would be better option, Since we gonna use the tasks only for Location App.
And Since this is only for location app then the file should be in **location folder instead of users**.